docs/en/ingest-management/release-notes/release-notes-8.17.asciidoc

```diff
@@ -26,6 +27,49 @@ Also see:
 
 
 
+// begin 8.17.4 relnotes
+
+[[release-notes-8.17.4]]
+== {fleet} and {agent} 8.17.4
+
+Review important information about the  8.17.4 release.
+
+[discrete]
+[[new-features-8.17.4]]
+=== New features
+
+The 8.17.4 release adds the following new and notable features.
+
+{agent}::
+* Add motel sample configurations. {agent-pull}6630[#6630] 
```

Comment:
```suggestion
* Add managed OTLP endpoint sample configurations. {agent-pull}6630[#6630] 
```

MOtel is an internal name that we don't want in external docs.

Reply:
Thanks! I had no idea.